File: apps/s-2-admin/src/api/workflowReminder/WorkflowReminderCreateInput.ts
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";
import { WorkflowStepWhereUniqueInput } from "../workflowStep/WorkflowStepWhereUniqueInput";

export type WorkflowReminderCreateInput = {
  booking?: BookingWhereUniqueInput | null;
  method: "EMAIL" | "SMS";
  scheduledDate: Date;
  referenceId?: string | null;
  scheduled: boolean;
  workflowStep: WorkflowStepWhereUniqueInput;
};

File: apps/cactus-demo-dotnet/src/APIs/WorkflowReminder/Dtos/WorkflowReminderCreateInput.cs
using CactusDemoDotnet.Core.Enums;

namespace CactusDemoDotnet.APIs.Dtos;

public class WorkflowReminderCreateInput
{
    public int? Id { get; set; }

    public Booking? Booking { get; set; }

    public MethodEnum Method { get; set; }

    public DateTime ScheduledDate { get; set; }

    public string? ReferenceId { get; set; }

    public bool Scheduled { get; set; }

    public WorkflowStep WorkflowStep { get; set; }
}
